plugins/gatsby-theme-iterative-docs/package.json

```diff
@@ -4,8 +4,31 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint-ts": "tsc --noEmit --skipLibCheck && eslint --ext .json,.js,.ts,.tsx ."
   },
```

Comment:
`lint-ts` helps ensure none of the packages were missed and nothing else ended up broken.